src/Components/ServiceScene/Breakdowns/SortByScene.tsx

```diff
@@ -21,16 +21,39 @@ export class SortCriteriaChanged extends BusEventBase {
 export class SortByScene extends SceneObjectBase<SortBySceneState> {
   public sortingOptions = [
     {
-      value: 'changepoint',
-      label: 'Most relevant',
-      description: 'Smart ordering of graphs based on the data',
+      label: '',
+      options: [
+        {
+          value: 'changepoint',
+          label: 'Most relevant',
+          description: 'Smart ordering of graphs based on the data',
```

Comment:
How about `Smart ordering of graphs based on the most significant spikes in the data'?

Reply:
Sounds good to me 👍